src/spryker/helm/preview/templates/nginx-service.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    kompose.cmd: kompose convert -c
+    kompose.version: 1.18.0 ()
+  creationTimestamp: null
+  labels:
+    io.kompose.service: nginx
+  name: nginx
+spec:
+  type: LoadBalancer
```

Comment:
This will assign an IP address when on GKE/AWS. As you already have an ingress object, set this to ClusterIP or omit it to use the default.